### ISSUE TITLE
Spamming retry hotkey while clicking interface button crashes game

### ISSUE THREAD

**Issue by Szago** [ppy/osu#12821]
**Describe the bug:**  Every time when I spam click my retry hotkey while clicking any other button, the game just instantly crashes. Tested: entering singleplayer button, playlists start button, settings button, entering a beatmap

**osu!lazer version:**  2021.514.0

**Logs:** 
[database.log](https://github.com/ppy/osu/files/6483628/database.log)
[network.log](https://github.com/ppy/osu/files/6483629/network.log)
[performance.log](https://github.com/ppy/osu/files/6483631/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/6483632/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/6483633/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/6483634/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/6483636/runtime.log)
[updater.log](https://github.com/ppy/osu/files/6483637/updater.log)

**Comment by frenzibyte** [comment]
Logs clean, I don't see the crash, reproduce and attach before starting again.

**Comment by frenzibyte** [comment]
Actually reproduced myself and as I figured, another duplicate of https://github.com/ppy/osu/issues/11511.
